cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
```

Comment:
Please use **weak**, and make it optional to avoid retain cycle, also remember that _**PaymentViewControllerDelegate**_ should be declared as **_AnyObject_**